### PR TITLE
{devel}[GCCcore/11.3.0] xtrans v1.3.5

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/x/xtrans/xtrans-1.3.5-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+name = 'xtrans'
+version = '1.3.5'
+
+homepage = "http://www.freedesktop.org/wiki/Software/xlibs"
+description = """xtrans includes a number of routines to make X implementations transport-independent;
+ at time of writing, it includes support for UNIX sockets, IPv4, IPv6, and DECnet.
+"""
+
+# toolchain = {'name': 'intel', 'version': '2016a'}
```
Comment (jfgrimm) [inline easybuild/easyconfigs/x/xtrans/xtrans-1.3.5-GCCcore-11.3.0.eb:11]:
```suggestion
```

File: easybuild/easyconfigs/x/xtrans/xtrans-1.3.5-GCCcore-11.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [XORG_LIB_SOURCE]
+
+checksums = ['b7a577c1b6c75030145e53b4793db9c88f9359ac49e7d771d4385d21b3e5945d']
+
+builddependencies = [
+    ('xorg-macros', '1.19.3'),
+    ('libxslt', '1.1.34'),
+]
```
Comment (jfgrimm) [inline easybuild/easyconfigs/x/xtrans/xtrans-1.3.5-GCCcore-11.3.0.eb:22]:
```suggestion
builddependencies = [
    ('binutils', '2.38'),
    ('xorg-macros', '1.19.3'),
    ('libxslt', '1.1.34'),
]
```

**Review by jfgrimm — CHANGES_REQUESTED** [review]

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen2

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in total)
node142.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/ec696548e82bda35c4601b375652fb87 for a full test report.

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node142.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/5ef5ab0e6a718a5b7d5e14d7c80924b0 for a full test report.

**Review by jfgrimm — APPROVED** [review]
LGTM

**Comment by jfgrimm** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**